tests/integration/services.test.js

```diff
@@ -38,40 +42,67 @@ describe.skip('UserMergeService', function () {
 
       await UserMergeService.mergeAccounts(ogUser1, ogUser2)
 
-      // TODO: get from db
-      const newParent = ogUser2
-      const [newChild1, newChild2, newChild3] = [ogUser1, mergedUser1, mergedUser2]
+      const newParent = getUser(ogUser2)
+      const newChild1 = getUser(ogUser1); const newChild2 = getUser(mergedUser1); const newChild3 = getUser(mergedUser2)
       expect(newChild1.mergeduserid).to.equal(newParent.userid)
       expect(newChild2.mergeduserid).to.equal(newParent.userid)
       expect(newChild3.mergeduserid).to.equal(newParent.userid)
       expect(newParent.mergeduserid).to.be.null()
     })
+    it('should correctly merge friends of an OG (parent) account into another parent account', async () => {
+      const ogUser1 = 1
+      const ogUser2 = 2
+      const friend1 = 11
+      // ogUser1 is being merged to ogUser2. friend1 is origionally a friend of ogUser1.
+
+      await insertUser({ userid: ogUser1, mergeduserId: null })
+      await insertUser({ userid: ogUser2, mergeduserId: null })
+      await insertUser({ userid: friend1, mergeduserId: null })
+
+      await addFriend(ogUser1, friend1)
+      await UserMergeService.mergeAccounts(ogUser1, ogUser2)
+
+      const newParentFriendship = checkFriendshipExists(ogUser2, friend1)
+      const newChildFriendship = checkFriendshipExists(ogUser1, friend1)
+      expect(newParentFriendship).to.be.equal(true)
+      expect(newChildFriendship).to.be.equal(false)
```

Comment:
The problem with using `testUtils` like this is that you end up testing code (`testUtils.checkFriendshipExists`) that don't actually match what is used in production (`UserService.login`). We should write our tests such that they align with what the users sees as much as possible. 

Checking whether `UserService.login` returns the correct friendships could potentially catch way more errors.

Reply:
Fair point